backend/fleet_management/models.py

```diff
@@ -125,3 +127,17 @@ def hash_form(initial_data: dict) -> int:
             "endMileage": initial_data["end_mileage"],
         }
         return hash_dict(required_fields)
+
+
+class Refuel(models.Model):
+    driver = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="drives_refueled"
+    )
+    car = models.ForeignKey(Car, null=False, on_delete=models.CASCADE)
+    date = models.DateField(auto_now_add=True, blank=False)
+    current_mileage = models.IntegerField(null=False)
+    refueled_liters = models.IntegerField(null=False)
+    price_per_liter = models.IntegerField(null=False)
```

Comment:
```suggestion
    current_mileage = models.PositiveIntegerField(null=False)
    refueled_liters = models.PositiveIntegerField(null=False)
    price_per_liter = models.PositiveIntegerField(null=False)
```
https://docs.djangoproject.com/en/3.0/ref/models/fields/#positiveintegerfield

None of the above fields can be smaller than zero but someone could input them as such in Django admin, which could mess data exports a bit.